### ISSUE TITLE
0.9c Reset freezes on Genesis games

### ISSUE THREAD

**Issue by snessy7** [ClusterM/retroarch-clover#93]
Hitting reset button while running Genesis games freezes unit.  All buttons as well as power switch unresponsive, had to pull USB cable.  Also when launching a Genesis save state buttons were unresponsive and system would not reset.  Did not test with other emulators.  Reverting to 0.9b resolved for now.

**Comment by ClusterM** [comment]
Upload please. There was broken version for about 5 minutes %)

**Comment by snessy7** [comment]
I just re-downloaded 0.9c and installed, reset lockup seems resolved, but loading existing Genesis save states results in a frozen unresponsive game screen with no sound.  Can reset from there however.  

Here is save state file for original Sega Genesis Road Rash.  Reverting to 0.9b for now.

[Road Rash (Sega Genesis).zip](https://github.com/ClusterM/retroarch-clover/files/1374361/Road.Rash.Sega.Genesis.zip)

**Comment by snessy7** [comment]
I tried with 1.0.  Old Genesis save states still would not load.  Also tried a Master System save state and it loaded with extremely distorted audio, although launching the game directly worked fine.  However, when creating new save states with 1.0 they loaded fine, so I think the save states I created with older versions were corrupt or incompatible.  These were also the save states that had black/blank preview images.  Will probably just delete the few i have as the new ones seem to be working.  Other 1.0 upgrades are really nice, everything working perfectly for me now across SNES/NES/GB/Genesis/SMS games.
